components/log-viewer-webui/server/src/S3Manager.js

```diff
@@ -55,4 +69,8 @@ class S3Manager {
     }
 }
 
-export default S3Manager;
+export default fastifyPlugin(async (app, region) => {
+    console.log(`Region in Plugin init is ${region}`)
+    console.log(region)
+    await app.decorate("s3Manager", new S3Manager(region));
```

Comment:
I think we can directly pass all options to the `S3Manager` constructor, assuming we will be adding more options in the future.
```suggestion
    await app.decorate("s3Manager", new S3Manager(options));
```